### PR TITLE
HOWTO-run_tests - added link to tips in getting postgres installed

### LINKED ISSUE #1208: Feature Request: Documentation -- test enviornment
**Is your feature request related to a problem? Please describe.**
When setting up my dev environment, it wasn't immediately clear how to run tests since the link was near the bottom of the page.  It also wasn't clear how to get Postgres up and running or how to debug problems with its installation. 

**Describe the solution you'd like**

* Add the testing section under *getting the app started* 
* can just link to the page [how to run tests](https://github.com/MoveOnOrg/Spoke/blob/main/docs/HOWTO-run_tests.md)
* in that page, include[ tips in getting postgres installed](https://www.codementor.io/engineerapart/getting-started-with-postgresql-on-mac-osx-are8jcopb)

### PR DESCRIPTION
**Pull request by tanyastickles**
# Fixes [#1208 ](https://github.com/MoveOnOrg/Spoke/issues/1208)

## Description

README: Links to the testing documentation more prominently. 
How to run tests: 
* Adds documentation for troubleshooting Prostgres installation. 
* Changes references from 'npm' to 'yarn'  e.g. yarn test-sqlite
* Added section for `yarn test-rediscache`

README:
Screen Shot 2019-11-17 at 9 07 57 PM

How to Run tests:
Screen Shot 2019-11-24 at 4 02 15 PM

# Checklist:

- [x] I have manually tested my changes on desktop and mobile
- [x] The test suite passes locally with my changes
- [x] If my change is a UI change, I have attached a screenshot to the description section of this pull request
- [x] [My change is 300 lines of code or less](https://github.com/MoveOnOrg/Spoke/blob/main/CONTRIBUTING.md#your-first-code-contribution), or has a documented reason in the description why it’s longer
- [x] I have made any necessary changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] My PR is labeled [WIP] if it is in progress

### CODE REVIEW

**Review by schuyler1d — DISMISSED** [review]
Thanks so much!

Since you're updating HOWTO-run_tests could you replace references to `npm...` to `yarn....` e.g. `yarn test-sqlite` -- for extra bonus, could you add `yarn test-rediscache`?

**Comment by tanyastickles** [comment]
sure thing! @schuyler1d 
also, where do we want to add the `yarn test-rediscache`?  Is it its own section or part of end-to-end testing?

**Comment by schuyler1d** [comment]
I think it's fine as its own section (or under postgres) -- whichever you think is clearer, but please add a line that says it's enabling and using Redis for caching and make it clear that it's not a different backend (i.e. sqlite vs postgres are different DB backends, but Redis cache testing defaults to postgres and is a sort of 'addon' to the DB for improved speed/scalability)

File: docs/HOWTO-run_tests.md [hunk 1]
```diff
@@ -2,23 +2,27 @@ There are current two ways to run tests, using either PostgreSQL or SQLite.
 
 ## PostgreSQL Testing (default, closer to most prod environments)
 
-1) Install PostgreSQL
+1) Install PostgreSQL - [Tips to installing Postgre](https://www.codementor.io/engineerapart/getting-started-with-postgresql-on-mac-osx-are8jcopb)
```
Comment (ibrand) [inline docs/HOWTO-run_tests.md]:
```suggestion
1) Install PostgreSQL - [Tips to installing Postgres](https://www.codementor.io/engineerapart/getting-started-with-postgresql-on-mac-osx-are8jcopb)
```

Comment (ibrand) [reply docs/HOWTO-run_tests.md]: Also thank you for this! Postgres can be a pain to install on Mac.

File: docs/HOWTO-run_tests.md [hunk 2]
```diff
@@ -2,23 +2,27 @@ There are current two ways to run tests, using either PostgreSQL or SQLite.
...
-3) Run `npm test`
+3) Run `yarn test`
 
 ## SQLite Testing (simpler)
 
-1) Run `npm run test-sqlite`
+1) Run `yarn run test-sqlite`
+  
+## Test Redis Cache
+
```
Comment (ibrand) [inline docs/HOWTO-run_tests.md:19]:
```suggestion
Redis is used for caching and is separate from the backend DB so can be used with sqlite *or* postgres. Redis cache testing defaults to postgres and functions like an 'addon' to the DB for improved speed/scalability.
```
Maybe add this sentence here right under this heading to clarify this section? Thanks for adding this!

**Review by ibrand — DISMISSED** [review]
Left a few suggestions, but overall I'm really into this. Thank you for writing this up! I've seen lots of people not even think to run the tests because we don't have this section in the docs, so this is a great improvement

File: docs/HOWTO-run_tests.md (hunk 2 above)
Comment (tanyastickles) [reply docs/HOWTO-run_tests.md:19]: thanks for the suggestion!

**Review by ibrand — APPROVED** [review]
🚀 ⭐️

**Pull request merged**